pkg/build/builder/daemonless.go

```diff
@@ -267,9 +248,6 @@ func pushDaemonlessImage(sc types.SystemContext, store storage.Store, imageName
 	}
 
 	systemContext := sc
-	// if credsDir, ok := os.LookupEnv("PUSH_DOCKERCFG_PATH"); ok {
-	// 	systemContext.AuthFilePath = filepath.Join(credsDir, ".docker", "config.json")
-	// }
 	systemContext.AuthFilePath = "/tmp/config.json"
```

Comment:
@gabemontero did we miss adding the same logic for image push? Granted, pushing to a mirror feels like a bad use case to support.

Reply:
It got discussed @adambkaplan and was deemed unnecessary.

Reply:
In the end I decided to minimize the moving parts

Reply:
for reference:  https://github.com/openshift/builder/pull/102#discussion_r328924879